### DISCUSSION TITLE
有没有好玩的mc整合包推荐

### DISCUSSION THREAD

**Discussion by Nirvana341** [share121/inter-knot#1797] [其他]
想找风景特别好，偏养老的整合包，绳匠们有没有推荐

**Comment by WGSheshang** [comment]
11111111111111111111

**Comment by WGSheshang** [comment]
**

> _111111111111111111111111111111111_

**

**Comment by ShellShield** [comment]
烂柯人？说是准备发整合包2.0了

**Comment by QingMu0314** [comment]
乌托邦，地球三，都是比较热门的，就是对性能要求比较大

**Reply by Nirvana341** [reply]
> 乌托邦，地球三，都是比较热门的，就是对性能要求比较大

试了试乌托邦，电脑带不动qwq
